client-ts/signalr/src/HttpConnection.ts

```diff
@@ -83,8 +83,9 @@ export class HttpConnection implements IConnection {
                 let headers;
                 const token = this.options.accessTokenFactory();
                 if (token) {
-                    headers = new Map<string, string>();
-                    headers.set("Authorization", `Bearer ${token}`);
+                    headers = {
```

Comment:
What is this crazy syntax? inline dictionary literals? Looks like C#

Reply:
There is no such thing as a "Dictionary" in JavaScript. This syntax is just a way of setting a property with a name that isn't a valid identifier. This code is functionally equivalent to the following:

```javascript
headers = {
    Authorization: `Bearer ${token}`,
}
```

However, since we're treating this object like a "dictionary" I wanted to use the `[]` syntax to make it clear. The `[]` syntax would only really be needed if we wanted to set a key with a name like `Foo Bar` (with spaces) or something.

Reply:
See [New notations in ECMAScript 2015](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Object_initializer)

Reply:
object literal.